composer.json

```
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": ">= 7.2.0",
+        "php": "^7.2|^8.0",
```

I prefer this format:
```suggestion
        "php": "^7.2 || ^8.0",
```